co_simulation/validation/fsi_turek_FSI2/README.md

```diff
@@ -0,0 +1,49 @@
+# Turek benchmark - FSI2
```

Comment:
This file is basically copying&pasting the one I did for the `FSIApplication` examples. I'd create a hyperlink to it in here.